src/webview/MessageListWeb.js

```diff
@@ -15,15 +15,32 @@ import { base64Utf8Encode } from '../utils/encoding';
 export default class MessageListWeb extends Component<Props> {
   context: Context;
   props: Props;
+  isWebViewReady: boolean;
   webview: ?Object;
-  isReady: boolean;
   unsentMessages: WebviewInputMessage[] = [];
 
   static contextTypes = {
     styles: () => null,
     theme: () => null,
   };
 
+  /*
+  *
+  * Synchronizing setup with webview
+  *
+  */
+  sendTestPing = (): void => {
+    if (this.isWebViewReady) {
```

Comment:
I don't think this will ever be `true` on the first run.
If we were to stick with `setTimeout` you could do something like
```js
if (!this.isWebViewReady) {
  setTimeout(this.sendTestPing, 10);
}
```